<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textDialog</class>
 <widget class="QDialog" name="textDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>&amp;Text</string>
    </attribute>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>233</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="fontSizeSpinner">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Input</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>491</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="textPreview">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>-1</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>AbYz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxBold">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxItalic">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxStrike">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxUnderline">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>&amp;Effects</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBoxtextDialog">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
